examples/with-tailwindcss/postcss.config.js

```diff
@@ -0,0 +1,6 @@
+module.exports = {
```

Comment:
I think it would be better to show how to use Tailwind with the CLI instead of PostCSS, this can be another example for PostCSS usage.

Reply:
okay I'll make the changes